<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeThresholdDialog</class>
 <widget class="QDialog" name="ChangeThresholdDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>275</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Change Threshold For Search Condition</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>221</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QComboBox" name="cbSlowD_threshold_1">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>115</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>MACD</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>SlowK</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbMACD_threshold_1">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>25</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>RSI</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbRSI_threshold_1">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>55</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSlowK_threshold_1">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>85</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>SlowD</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>211</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Close-F/S</string>
     </property>
     <widget class="QComboBox" name="cbCLS_threshold_1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cbCLF_threshold_1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>55</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>C &gt; F</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbCGF_threshold_1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>25</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>C &lt; F</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>115</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>C &gt; S</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbCGS_threshold_1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>85</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>C &lt; S</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>SlowD</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbMACD_threshold_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>25</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>MACD</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>RSI</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbRSI_threshold_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>55</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSlowD_threshold_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>115</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>211</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Close-F/S</string>
     </property>
     <widget class="QComboBox" name="cbCLS_threshold_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cbCGF_threshold_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>25</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>C &lt; F</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbCLF_threshold_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>55</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>C &gt; F</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>115</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>C &gt; S</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbCGS_threshold_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>85</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>C &lt; S</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>SlowK</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSlowK_threshold_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>85</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Tab 3</string>
    </attribute>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>SlowD</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbMACD_threshold_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>25</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>MACD</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>RSI</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbRSI_threshold_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>55</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSlowD_threshold_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>115</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>211</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Close-F/S</string>
     </property>
     <widget class="QComboBox" name="cbCLS_threshold_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cbCGF_threshold_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>25</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>C &lt; F</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbCLF_threshold_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>55</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>C &gt; F</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>115</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>C &gt; S</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbCGS_threshold_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>85</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>C &lt; S</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>SlowK</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSlowK_threshold_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>85</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Tab 4</string>
    </attribute>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>SlowD</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbMACD_threshold_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>25</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>MACD</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>RSI</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbRSI_threshold_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>55</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSlowD_threshold_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>115</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>211</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Close-F/S</string>
     </property>
     <widget class="QComboBox" name="cbCLS_threshold_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cbCGF_threshold_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>25</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>C &lt; F</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbCLF_threshold_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>55</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>C &gt; F</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>115</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>C &gt; S</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbCGS_threshold_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>85</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>C &lt; S</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>SlowK</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSlowK_threshold_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>85</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Tab 5</string>
    </attribute>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>SlowD</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbMACD_threshold_5">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>25</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>MACD</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>RSI</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbRSI_threshold_5">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>55</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSlowD_threshold_5">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>115</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>211</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Close-F/S</string>
     </property>
     <widget class="QComboBox" name="cbCLS_threshold_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cbCGF_threshold_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_37">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>25</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>C &lt; F</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbCLF_threshold_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_38">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>55</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>C &gt; F</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_39">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>115</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>C &gt; S</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbCGS_threshold_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_40">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>85</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>C &lt; S</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>SlowK</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSlowK_threshold_5">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>85</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
